internal/services/bot/bot_service_resource_base.go

```diff
@@ -129,6 +129,12 @@ func (br botBaseResource) arguments(fields map[string]*pluginsdk.Schema) map[str
 			Default:  false,
 		},
 
+		"icon_url": {
```

Comment:
can we add this field [to the documentation](https://github.com/hashicorp/terraform-provider-azurerm/blob/b7eb5a3e8cecd273cd788f606aa46bd323cc68c0/website/docs/r/bot_service_azure_bot.html.markdown?plain=1#L75-L85) too?